services/traction/api/db/models/base.py

```diff
@@ -36,3 +36,20 @@ class BaseTable(BaseModel):
             TIMESTAMP, nullable=False, server_default=func.now(), onupdate=func.now()
         )
     )
+
+
+class TimestampModel(BaseModel):
+    created_at: datetime = Field(
+        sa_column=Column(TIMESTAMP, nullable=False, server_default=func.now())
+    )
+    updated_at: datetime = Field(
+        sa_column=Column(
+            TIMESTAMP, nullable=False, server_default=func.now(), onupdate=func.now()
+        )
+    )
+
+
+class StatefulModel(TimestampModel):
```

Comment:
might be better to leave these separate (both inherit from `BaseModel`) Then we can pick and choose.

Reply:
did that.